node-webservice/react-client/src/App.jsx

```diff
@@ -69,9 +69,11 @@ class App extends Component {
     return (
       <div className="App">
         <header className="App-header">
-          <img src='/logo.svg' alt="app logo" style={{margin:'0.5vmin 3vmin', width:'calc(30px + 5vmin)'}}/>
-          <h1 style={{textAlign:'center'}}>Build-a-Bot Royale</h1>
-          <img src='/logo.svg' alt="app logo" style={{margin:'0.5vmin 3vmin', width:'calc(30px + 5vmin)'}}/>
+          <span className="App-header-emoji" role="img" aria-label="image logo">⚡️</span>
+          <h1 className="App-title" style={{textAlign:'center'}}>
+            <span className="App-title-text-bkg">BuilD-a-BoT rOYaLE</span>
+          </h1>
+          <span className="App-header-emoji" role="img" aria-label="image logo">🤖</span>
```

Comment:
The change in style of the header is causing the webpage to become scrollable both horizontally and vertically, and the sizing is odd, defaulting to three lines, even when it should be able to stretch wider. 
![image](https://user-images.githubusercontent.com/28120258/76477737-7fac6b00-63c3-11ea-818e-a7aa4fbeee88.png)